client/src/stories/assets/buttonPlus.svg

```diff
@@ -0,0 +1,27 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="58" height="58" viewBox="0 0 58 58">
```

Comment:
There are some not used properties in your SVGs. These properties are created by design tools like Adobe XD.
For example, you can remove id and data-name.
Take a look at https://www.svgminify.com/

Similar to https://github.com/Hanttone/capstone-project/pull/8#discussion_r528816313